rhel/SPECS/burp.spec

```diff
@@ -1,10 +1,10 @@
 Name:		burp
 Summary:	Burp is a network-based simple yet powerful backup and restore program for Unix and Windows.
-Version:	2.0.36
+Version:	2.0.46
 Release:	1%{?dist}
 License:	GPL
 URL:		http://burp.grke.org/
-Source0:	https://github.com/grke/burp/archive/burp-%{version}.tar.gz
+Source0:	https://github.com/grke/burp/archive/burp-%{version}.tar.bz2
```

Comment:
Hello,
Is that correct?
I go to https://github.com/grke/burp/archive/burp-2.0.46.tar.bz2 and it doesn't exist.